package.json

```diff
@@ -35,27 +36,58 @@
     "fix": "npm run fix:lint && npm run fix:prettier",
     "start": "npm run develop",
     "serve": "gatsby serve",
-    "test": "echo \"Write tests! -> https://gatsby.app/unit-testing\"",
-    "commit": "git-cz"
+    "commit": "git-cz",
+    "cypress:open": "cross-env TZ=UTC node_modules/.bin/cypress open",
+    "cypress:run": "cross-env TZ=UTC node_modules/.bin/cypress run",
```

Comment:
You shouldn't need to prefix the `cypress` executable with `node_modules/.bin` since you're using `npm` or `yarn`, the package managers would auto pick the `node_modules` path for you.

You don't have to remove this, it's just a suggestion that the `node_module` path prefix isn't compulsory to run this commands.

Reply:
Would be taken care of during clean up.